concreep-refilled/control.lua

```diff
@@ -64,15 +66,20 @@ function init()
 	global.reinit_current_surface_roboport_index = 1
 	global.reinit_level_running = 0
 
-	for each, surface in pairs(game.surfaces) do
-		local roboports = surface.find_entities_filtered{type="roboport"}
-		for index, port in pairs(roboports) do
-			if is_valid_roboport(port) then
-				add_creeper_for_roboport(port)
-			end
-		end
-	end
-end
+	   for str in SURFACES:gmatch("([^,]+)") do

```

Comment:
Yep, and the reinit running right away is fine, because it kickstarts things in a UPS friendly way, as opposed to how the init was doing it. Haven't decided what to rename reinit to though.